topics/variant-analysis/tutorials/microbial-variants/tutorial.md

```diff
@@ -0,0 +1,235 @@
+
+- mutant_R1.fastq & mutant_R2.fastq - the read files in fastq format.
+- wildtype.fna - The sequence of the reference strain in fasta format.
+- wildtype.gbk - The reference strain with gene and other annotations in genbank format.
+- wildtype.gff - The reference strain with gene and other annotations in gff3 format.
+
+This data is available at Zenodo using the following [link](https://doi.org/10.5281/zenodo.582600).
+
+> ### {% icon hands_on %} Hands-on: Get the data
+>
+> 1.  Import all of the following files into a new history:
+>     - [mutant_R1.fastq](https://zenodo.org/record/582600/files/mutant_R1.fastq)
+>     - [mutant_R2.fastq](https://zenodo.org/record/582600/files/mutant_R2.fastq)
+>     - [wiltype.fna](https://zenodo.org/record/582600/files/wildtype.fna)  
+>     - [wiltype.gbk](https://zenodo.org/record/582600/files/wildtype.gbk)
+>     - [wiltype.gff](https://zenodo.org/record/582600/files/wildtype.gff)
+>
+>    > ### {% icon tip %} Tip: Importing data via links
+>    >
+>    > * Copy the link location
+>    > * Open the Galaxy Upload Manager
+>    > * Select **Paste/Fetch Data**
+>    > * Paste the link into the text field
+>    > * Press **Start**
+>    {: .tip}
+>
+{: .hands_on}
+
+Your history pane should look like this after you have uploaded all of the data:
+
+![micro-start-history.png](../../images/micro-start-history.png)
+
+# Section 2: Find variants with Snippy
+
+We will now run the Snippy tool on our reads, comparing it to the reference. If we give Snippy an annotated reference, it will silently run a tool called snpeff which will figure out the effect of any changes on the genes and other features. If we just give Snippy the reference sequence alone without the annotations, it will not run snpeff.
+
+We have an annotated reference and so will use it in this case.
+
+> ### {% icon hands_on %} Hands-on: Run Snippy
+>
+> 1. Go to the Galaxy tools panel, and use the search box at the top to search for **Snippy** and then open it's interface.
+> 1. For **Reference type** select *Genbank*.
+> 1. Then for **Reference Genbank** choose the *wildtype.gbk* file.
+> 1. For **Single or Paired-end reads** choose *Paired*.
+> 1. Then choose the first set of reads, *mutant_R1.fastq* and second set of reads, *mutant_R2.fastq*.
+> 1. For **Cleanup the non-snp output files** select *No*.
+> <br/>
+> <br/>
+> Your tool interface should look like this:
+>
+>  ![Snippy interface](../../images/interface.png)
+>
+> - Click **Execute**.
+>
+{: .hands_on}
+
+## Step 3: Examine Snippy output
+
+Snippy has taken the reads, mapped them against the reference using BWA MEM, looked through the resulting BAM file and found differences using some fancy Bayesian statistics (Freebayes), filtered the differences for sensibility and finally checked what effect these differences will have on the predicted genes and other features in the genome.
+
+It produces quite a bit of output, there can be up to 10 output files.
+
+| Filename | Description  |
```

Comment:
What about simplifying the table? 

```
Filename | Description
--- | ---
snps vcf file | The final annotated variants in VCF format
snps gff file | The variants in GFF3 format
snps table | A simple tab-separated summary of all the variants
snps summary  | A summary of the snps called
log file  | A log file with the commands run and their outputs
aligned fasta | A version of the reference but with - at position with depth=0 and N for 0 < depth < --mincov **(does not have variants)** 
consensus fasta | A version of the reference genome with all variants instantiated
mapping depth | A table of the mapping depth
mapped reads bam  | A bam file containing all of the mapped reads
outdir  | A tarball of the Snippy output directory for inout into Snippy-core if required
```

I removed the separation between the lines, and the `|` at the beginning and end of the lines. I should render similarly on the website but simplify the markdown.
![screen shot 2018-01-23 at 14 28 15](https://user-images.githubusercontent.com/1842467/35278252-ac5e2a2e-0049-11e8-8ce1-1b3a6276876a.png)
What do you think?